assests/javascript/index.js

```diff
@@ -44,6 +51,68 @@ async function addTask(task) {
   }
 }
 
+async function getAllTasks() {
+  try {
+    const response = await fetch(`${baseUrl}/todos`);
+    const {data} = await response.json();
+
+    return sortTasks(data);
+  } catch {
+    showConnecionErrorMassage();
+  }
+}
+
+async function showAllTasks() {
+  const tasks = await getAllTasks();
+
+  taskList.innerHTML = tasks.map((task) => {
+    return `<li class="task-box tooltip" id=${task.id}>
+              <input type="checkbox" class="task-check-box" ${isChecked(task.isDone)}></input>
+              <p class="task-title">${task.text}</p>
+              <i class="fa-solid fa-pen-to-square edit-task-icon"></i>
+              <i class="fa-solid fa-xmark delete-task-icon"></i>
+
+              <p class="tooltiptext">${convertTime(task.createdAt)}</p>
+            </li>`;}).join("");
```

Comment:
I'm not sure if editor is formatting your code correctly or not. But a new line after `;` would be nice.

Reply:
done